src/main/java/Duke.java

```diff
@@ -1,10 +1,238 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
+    static final String SECTION_DIVIDER = "____________________________________________________________";
+    private static void greetUser(){
+        String greeting = "\t" + SECTION_DIVIDER + "\n"
+                + "\tHello! I'm Duke. \n"
+                + "\tWhat can I do for you? \n"
+                + "\t" + SECTION_DIVIDER;
+        System.out.println(greeting);
+    }
+    private static void signOff(){
+        String sign_off = "\t" + SECTION_DIVIDER + "\n"
+                + "\tBye. Hope to see you again soon!\n"
+                + "\t" + SECTION_DIVIDER;
+        System.out.println(sign_off);
+    }
+    private static final String COMMAND_WORDS = "list" + "done" + "todo" + "deadline" + "event" + "delete";
+
+    static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void addTask(String taskDescription) throws
```

Comment:
This method is on the longer side, which is [not recommended](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#avoid-long-methods) in terms of code quality. You may consider revising this such that details could be abstracted away.